### DISCUSSION TITLE
Can connect Rowy Run to GCP

### DISCUSSION THREAD

**Discussion by ozan-celebi** [AntlerVC/firetable#1330] [Support / Q&A] [answered]
Hello,

I cant connect my GCP to Rowy Run. I am the creator of the GCP project but I get the following error:

<img width="601" alt="image" src="https://github.com/rowyio/rowy/assets/138249319/fddcbce1-0765-4a15-a966-eb793ceeb4f3">

**Comment by BeeBombshell** [comment, marked as answer]
Hey @ozan-celebi, could you check if this is the same account you've used to set up your project? Also, if this is the case, it might as well be an issue of the organizational restrictions and settings on GCP. Could you try following the steps as mentioned [here](https://docs.rowy.io/troubleshooting/project-creation-issue) and see if it fixes it?
